package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "watch": "turbo run watch --filter=!pie-monorepo",
-    "build": "turbo run build --filter=!pie-monorepo",
+    "build": "turbo run build --filter=!pie-monorepo --filter=!wc-react18",
```

Comment:
I think this might get out of sync easily when we add more apps like angular, vue, and next etc.., is there a way to exclude anything in the app folder? @siggerzz might have ideas here 💯

Reply:
wasn't too difficult in the end - turns out you can filter by directory 🎉